global.json

```diff
@@ -0,0 +1,11 @@
+{
+  "tools": {
+    "dotnet": "3.0.100-preview-009750"
+  },
```

Comment:
What does this do? I just needed `"sdk"`

Reply:
Uhh, [that's not true](https://github.com/aspnet/AspNetCore-Tooling/blob/master/global.json#L3)?